sklearn/semi_supervised/label_propagation.py

```diff
@@ -358,6 +362,18 @@ class distributions will exceed 1 (normalization may be desired).
             affinity_matrix /= normalizer[:, np.newaxis]
         return affinity_matrix
 
+    def fit(self, X, y):
+        # alpha is deprecated for LabelPropagation because it doesn't have any
+        # theoretical meaning (from the reference paper). Look at PR 6727.
+        if self.alpha is not None:
```

Comment:
To follow your suggestion and pass the tests, I would need to change the `__init__` of `BaseLabelPropagation`, which is probably not what we want since `LabelSpreading` can really use different `alpha` values.

This problem was discussed with @amueller before, in his code review.